<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioVerdeTube</class>
 <widget class="QDialog" name="RadioVerdeTube">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>469</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>469</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>469</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RadioVerdeTube</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>435</width>
       <height>469</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>435</width>
       <height>469</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_superior">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
background-color:rgb(0,0,0);
}
QPushButton{
background-color:#000000ff;
border-radius:20px;
}
QPushButton:hover{
background-color:rgb(53,53,79);
border-radius:20px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255,222,222)</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../media/radTub.svg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font:87 12pt &quot;Arial Black&quot;;
color:rgb(20,200,220);</string>
           </property>
           <property name="text">
            <string>RadioVerdeTube</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>12</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnInfo">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../media/info.svg</normaloff>../media/info.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>38</width>
             <height>38</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_min">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../media/minimizar.svg</normaloff>../media/minimizar.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>38</width>
             <height>38</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../media/cerrar.svg</normaloff>../media/cerrar.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>38</width>
             <height>38</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_medio">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_input">
           <property name="styleSheet">
            <string notr="true">QFrame{
background-color:rgb(53,53,79);
}

QLabel{
font:97 12pt 'Arial Black';
color:rgb(20,200,220);
border:0px solid #14C8DC;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QFrame" name="frame_input1">
              <property name="styleSheet">
               <string notr="true">QLabel{
font:97 12pt 'Arial Black';
color:rgb(20,200,220);
border:0px solid #14C8DC;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="styleSheet">
                  <string notr="true">color:rgb(20,200,220);</string>
                 </property>
                 <property name="text">
                  <string>Url del Video:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineUrlVideo">
                 <property name="styleSheet">
                  <string notr="true">background-color:rgb(0,0,0);
color:rgb(200,200,220);
font: 75 italic 10pt &quot;Verdana&quot;;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_input1_2">
              <property name="styleSheet">
               <string notr="true">QLabel{
font:97 12pt 'Arial Black';
color:rgb(20,200,220);
border:0px solid #14C8DC;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">color:rgb(20,200,220);</string>
                 </property>
                 <property name="text">
                  <string>Nombre del Archivo:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineNameFile">
                 <property name="styleSheet">
                  <string notr="true">background-color:rgb(0,0,0);
color:rgb(200,200,220);
font: 75 italic 10pt &quot;Verdana&quot;;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_input1_3">
              <property name="styleSheet">
               <string notr="true">QLabel{
font:97 12pt 'Arial Black';
color:rgb(20,200,220);
border:0px solid #14C8DC;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="styleSheet">
                  <string notr="true">color:rgb(20,200,220);</string>
                 </property>
                 <property name="text">
                  <string>Artista</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineArtist">
                 <property name="styleSheet">
                  <string notr="true">background-color:rgb(0,0,0);
color:rgb(200,200,220);
font: 75 italic 10pt &quot;Verdana&quot;;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="styleSheet">
                  <string notr="true">color:rgb(20,200,220);</string>
                 </property>
                 <property name="text">
                  <string>Formato de Salida</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboFormat">
                 <property name="styleSheet">
                  <string notr="true">color:rgb(200,200,220);
font: 75 italic 12pt &quot;Verdana&quot;;
background-color:rgb(0,0,0);</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>.mp3</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>.wav</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>.ogg</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushClear">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color:rgb(20,200,220);
border: 1px solid white;
border-radius:5px;
}
QPushButton:hover{
background-color:black;
}</string>
                 </property>
                 <property name="text">
                  <string>Limpiar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_inferior">
        <property name="styleSheet">
         <string notr="true">QFrame{
background-color:rgb(53,53,79);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_btns">
           <property name="styleSheet">
            <string notr="true">QFrame{
background-color:rgb(53,53,97);
}
QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color:rgb(20,200,220);
border: 1px solid white;
border-radius:5px;
}
QPushButton:hover{
background-color:black;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QPushButton" name="pushSelectDir">
              <property name="text">
               <string>Seleccionar Carpeta</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushDownload">
              <property name="text">
               <string>Descargar Archivo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushOpen">
              <property name="text">
               <string>Mostar Archivo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>10</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color:rgb(250,230,23);
font: 75 9pt &quot;Nirmala UI&quot;;</string>
              </property>
              <property name="text">
               <string>Desarrollado Por: Emilio Lucero Loza</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_img">
           <property name="styleSheet">
            <string notr="true">background-color:black;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QProgressBar" name="pbDownload">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>RadioVerdeTube</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_min</sender>
   <signal>clicked()</signal>
   <receiver>RadioVerdeTube</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushClear</sender>
   <signal>clicked()</signal>
   <receiver>lineUrlVideo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushClear</sender>
   <signal>clicked()</signal>
   <receiver>lineNameFile</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushClear</sender>
   <signal>clicked()</signal>
   <receiver>lineArtist</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
